FAQ.md

```diff
@@ -1,5 +1,9 @@
 # Frequently Asked Questions
 
+### Troubles with the installation process?
```

Comment:
You should add *on Windows* (the xda video explains only for Windows).

Also, there is no installation involved. Maybe "Troubles to make it work on Windows?" or something similar and better. What do you think?